styleguide/source/_patterns/02-molecules/article-teaser--extended.twig

```diff
@@ -0,0 +1,37 @@
+{% set teaserStyle =  teaser.style ? "joe__article-teaser--" ~ teaser.style : "" %}
+
+
+<article class="joe__article-teaser {{teaserStyle}}">
+  {% if teaser.image %}
+  <a href="{{teaser.title.href}}" class="joe__article-teaser__image">
+    {% set image = teaser.image %}
+    {% include "@atoms/image.twig" %}
+  </a>
+  {% endif %}
+  <div class="joe__article-teaser__block">
+    {% set flag = teaser.flag %}
+    {% include "@atoms/flag.twig" %}
+    {% set link = teaser.title %}
+    <h2 class="joe__article-teaser__title">{% include "@atoms/link.twig" %}</h2>
+    <span class="joe__article-teaser__meta">
+      {% for author in teaser.authors %}
```

Comment:
I wonder if it would make sense to make an author atom so you don't have to repeat this logic elsewhere.